code/weapon/beam.cpp

```diff
@@ -1177,6 +1167,39 @@ void beam_move_all_post()
 			}
 		}
 
+		// deal with ammo/energy for fighter beams
+		bool multi_ai = MULTIPLAYER_CLIENT && (moveup->objp != Player_obj);
+		bool cheating_player = Weapon_energy_cheat && (moveup->objp == Player_obj);
+		if (moveup->flags & BF_IS_FIGHTER_BEAM && !multi_ai && !cheating_player) {
```

Comment:
Just add a line here to help coders see the start of the if block, and I think we're good.